client/boot/index.js

```diff
@@ -323,7 +323,11 @@ function reduxStoreReady( reduxStore ) {
 			} );
 
 			if ( '/' === context.pathname && config.isEnabled( 'devdocs/redirect-loggedout-homepage' ) ) {
-				page.redirect( '/devdocs/start' );
+				if ( config.isEnabled( 'oauth' ) ) {
+					page.redirect( '/authorize' );
+				} else {
+					page.redirect( '/devdocs/start' );
```

Comment:
I think they are not exactly the same — regular oauth should work for users that don't want to develop, and devdocs/start should be for contributors. Perhaps `/authorize` can become a generic start and we have a link for devdocs from there.

cc @folletto for thoughts.